deploy/stacks/cdk_nag_exclusions.py

```diff
@@ -28,6 +28,10 @@
         'id': 'AwsSolutions-CB3',
         'reason': 'Access to docker daemon is required to build docker images',
     },
+    {
+        'id': 'AwsSolutions-SMG4',
+        'reason': 'Database is used for test purposes',
+    },
 ]
 
```

Comment:
Why do we get this exception in this PR?

Reply:
I think because I upgraded the aws-cdk-lib